docs/Golden-Gate/Bridges.rst

```diff
@@ -38,156 +24,161 @@ Use Cases
-#. The serialized constructor argument for that generator (e.g. firesim.bridges.UARTKey)
-#. A list of channel names; string references to Channel annotations
-
-And a series of FAMEChannelConnectionAnnotations, which target the module's I/O to group them into token channels.
+Finally at the bottom of your Bridge's class definition **you'll need to call generateAnnotations()**. This is necessary to have Golden Gate properly detect your bridge.
 
 You can freely instantiate your Bridge anywhere in your Target RTL: at the I/O
-boundary of your chi or deep in its module hierarchy.  Since all of the Golden
+boundary of your chip or deep in its module hierarchy.  Since all of the Golden
 Gate-specific metadata is captured in FIRRTL annotations, you can generate your
 target design and simulate it a target-level RTL simulation or even pass it off
 to ASIC CAD tools -- Golden Gate's annotations will simply be unused.
 
 What Happens Next?
 ------------------------
 
-If you do pass your FIRRTL & Annotations to Golden Gate. It will find your
-module, remove it,  and wire its dangling target-interface to the top-level of
-the design. During host-decoupling transforms, Golden Gate aggregates fields of
-your bridge's target IO based on ChannelAnnotations, and wraps them up into
-new Decoupled interfaces that match your Host Interface definition. Finally,
-once Golden Gate is done performing compiler transformations, it iterates
-through each Bridge annotation, generates your Module, passing it the
-serialized constructor argument, and connects it to the tokenized interface
-presented by the now host-decoupled target.
-
-Host-side Implementation
-------------------------
+If you pass your design to Golden Gate, it will find your target-side module, remove it,
+and wire its dangling target-interface to the top-level of the design. During
+host-decoupling transforms, Golden Gate aggregates fields of your bridge's
+target interface based on channel annotations emitted by the target-side of
+your bridge, and wraps them up into decoupled interfaces that match your host
+interface definition. Finally, once Golden Gate is done performing compiler
+transformations, it generates the bridge modules (by looking up their
+constructors and passing them their serialized constructor argument) and
+connects them to the tokenized interfaces presented by the now host-decoupled simulator.
+
+Host Side
+---------
 
-Host-side implementations have two components.
-#. A FPGA-hosted BridgeModule.
-#. An optional, CPU-hosted, bridge driver.
+The host side of a bridge has two components:
 
-In general, bridges have both a module and a driver: in FASED memory timing
-models, the BridgeDriver configures timing parameters at the start of
+#. A FPGA-hosted bridge module (``BridgeModule``).
+#. An optional, CPU-hosted, bridge driver (``bridge_driver_t``).
+
+In general, bridges have both: in FASED memory timing
```

Comment:
You say that in general, bridges have both, but for the FASED model you then you only describe the role of the driver in this relationship.